kitsune/questions/cron.py

```diff
@@ -196,13 +198,15 @@ def report_employee_answers():
         Date: {date}
         Number of questions asked: {num_questions}
         Number of questions answered: {num_answered}
-        Number of questions with answer from 'Support Forum Tracked' users: {num_by_tracked}
         """)
     email_body = email_body_tmpl.format(
         date=day_before_yesterday,
         num_questions=num_questions,
-        num_answered=num_answered,
-        num_by_tracked=num_answered_by_tracked)
+        num_answered=num_answered)
+
+    for username, count in num_answered_by_tracked.items():
+        email_body += 'Number of questions answered by {username}: {count}\n'.format(
+            username=username, count=count)
```

Comment:
I was under the impression that this email was a temporary thing that should go away in the future. That's why I tried to keep it self contained. But you never know..